rero_ils/modules/items/api/circulation.py

```diff
@@ -68,6 +68,12 @@ def wrapper(item, *args, **kwargs):
             # we arent allowed to validate a request on items with no requests.
             raise NoCirculationAction(
                 'No circulation action is possible')
+        elif function.__name__ == 'extend_loan':
+            # we arent allowed to extend an item that is not checked out.
+            loan = item.get_first_loan_by_state(state=LoanState.ITEM_ON_LOAN)
+            if not loan:
```

Comment:
cosmetic: should be write in one line: `if not item.get_first_loan_by_state(state=LoanState.ITEM_ON_LOAN):`

Reply:
Problem that I am using the variable `loan` later if needed . So I will leave it like that